tsconfig.json

```diff
@@ -18,8 +19,14 @@
     "noImplicitAny": false,
     "noImplicitThis": false,
     "strictNullChecks": false,
-    "declaration": true
+    "declaration": true,
+    "baseUrl": "./",
+    "paths": {
+      "react": ["./node_modules/preact/compat/"],
```

Comment:
that's interesting, what's the logic behind this?

Reply:
I vaguely remember that was a short-term solution for a weird Preact/TypeScript bug. We should be able to remove this in the future.